PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/projects/HDInsightModuleBuilder.java

```diff
@@ -71,7 +71,6 @@ public String getBuilderId() {
         return "HDInsight";
     }
 
-    @Override
     public Icon getBigIcon() {
```

Comment:
Please make the decision on your side. Since it is hdinsight code. And we are not expert. But make sure the change could built on both 2017.1, 2017.2